src/fluree/db/commit/storage.cljc

```diff
@@ -34,28 +34,37 @@
       [credential-subject commit-data])
     [commit-data nil]))
 
-(defn read-data-jsonld
-  [storage address]
-  (go-try
-    (let [jsonld (<? (storage/read-json storage address))]
-      (-> jsonld
-          (assoc "f:address" address)
-          json-ld/expand))))
-
-(defn read-commit-jsonld
+(defn read-verified-commit
```

Comment:
What does "verified" mean in this context?

Reply:
It means that the commit that is read has also been verified, and that the returned value includes the proof fro verification.

Reply:
Does that mean we re-hash and compare? I'm not sure we actually do that anywhere currently.